lib/push.js

```diff
@@ -78,7 +79,16 @@ var reasonToEvents = {
   }
 }
 
-module.exports = function (log, db) {
+module.exports = function (log, db, config) {
+
+  var vapid
+  if (config.vapidKeysFile) {
+    vapid = JSON.parse(fs.readFileSync(config.vapidKeysFile))
```

Comment:
You can also `require` JSON files directly, it returns the parsed result exactly like combining `JSON.parse` and `fs.readFileSync` does.